sklearnex/neighbors/_lof.py

```diff
@@ -162,6 +201,4 @@ def kneighbors(self, X=None, n_neighbors=None, return_distance=True):
         )
 
     fit.__doc__ = sklearn_LocalOutlierFactor.fit.__doc__
-    fit_predict.__doc__ = sklearn_LocalOutlierFactor.fit_predict.__doc__
-    predict.__doc__ = sklearn_LocalOutlierFactor.predict.__doc__

```

Comment:
Why you are not using it from sklearn?

Reply:
Available_if blocks it unfortunately. It's really a bad construction on sklearn's side